be/src/common/config.h

```diff
@@ -137,7 +140,7 @@ namespace config {
     // the maximum number of bytes to display on the debug webserver's log page
     CONF_Int64(web_log_bytes, "1048576");
     // number of threads available to serve backend execution requests
-    CONF_Int32(be_service_threads, "64");
+    CONF_Int32(be_service_threads, "512");
```

Comment:
I think 512 is some big, why not keep it 64?
For most scenario, 64 is enough.

Reply:
actually for 64, the query and stream load performance is poor, as mentioned in issue #3696, query and submit agent task share the same thread pool in thrift server. I have done some test, 512 can apparently improve the performance both for stream load and query(can not be rpc timeout), query would not by blocked by stream load.in this setting, the publish timeout almost disapper.

Reply:
sorry, I find this config make no sense when thrift server use threaded type , so I revert the modification